api/src/main/java/com/velocitypowered/api/proxy/config/ProxyConfig.java

```diff
@@ -84,7 +84,7 @@ public interface ProxyConfig {
    *
    * @return registered servers map
    */
-  Map<String, String> getServers();
+  Map<String, BackendServerConfig> getServers();
```

Comment:
This change breaks compatibility with current versions. It is better to create another method, for example `#getBackendServers()`, to obtain the configuration of each server and deprecate this method